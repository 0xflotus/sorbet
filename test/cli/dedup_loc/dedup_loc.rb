# typed: strict

begin
rescue Exception => e
    T.assert_type!(e, NilClass)
end
