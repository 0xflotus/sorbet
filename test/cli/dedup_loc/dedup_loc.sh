#!/bin/bash

main/sorbet test/cli/dedup_loc/dedup_loc.rb 2>&1
