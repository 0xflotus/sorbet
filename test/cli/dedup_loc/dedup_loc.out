test/cli/dedup_loc/dedup_loc.rb:5: Argument does not have asserted type `NilClass` http://go/e/7007
     5 |    T.assert_type!(e, NilClass)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  Got Exception originating from:
    test/cli/dedup_loc/dedup_loc.rb:4:
     4 |rescue Exception => e
     5 |    T.assert_type!(e, NilClass)
    test/cli/dedup_loc/dedup_loc.rb:4:
     4 |rescue Exception => e
               ^^^^^^^^^
Errors: 1
